src/virtualenv/seed/embed/base_embed.py

```diff
@@ -18,14 +20,21 @@ def __init__(self, options) -> None:
 
         self.pip_version = options.pip
         self.setuptools_version = options.setuptools
-        self.wheel_version = options.wheel
 
         self.no_pip = options.no_pip
         self.no_setuptools = options.no_setuptools
-        self.no_wheel = options.no_wheel
         self.app_data = options.app_data
         self.periodic_update = not options.no_periodic_update
 
+        if options.no_wheel:
+            warn(
```

Comment:
@stefanor I would also like to point out that as an user, I did not see this warning when I used the option.